bentoml/cli/bento_management.py

```diff
@@ -175,39 +172,60 @@ def list_bentos(
         _print_bentos_info(result, output)
 
     @cli.command(
-        help='Delete bento. To delete multiple bentos provide the name '
-        'version tag separated by "," for example "bentoml delete name:v1,name:v2',
+        help='Delete bentos. To delete a bento service use "--bento-name" and '
+        '"--bento-version" option: "bentoml delete --bento-name IrisClassifier '
+        '--bento-version 0.1.0". To delete multiple bentos, use "--bento-name" and/or '
+        '"--labels" to filter. To delete all bento services use "--all" option'
+    )
+    @click.option(
+        '--all', is_flag=True, help='Use this flag to remove all BentoServices'
+    )
+    @click.option(
+        '--labels',
+        type=click.STRING,
+        help="Label query to filter BentoServices, supports '=', '!=', 'IN', 'NotIn', "
+        "'Exists', and 'DoesNotExist'. (e.g. key1=value1, key2!=value2, key3 "
+        "In (value3, value3a), key4 DoesNotExist)",
     )
-    @click.argument("bentos", type=click.STRING, callback=parse_bento_tag_list_callback)
+    @click.option("--bento-name", type=click.STRING, help='BentoService name')
```

Comment:
could we make it just `--name` and `--version` here?

Reply:
Similarly to the repository API client, make this backwards compatible by supporting the bento_tag argument in CLI, so that existing CLI scripts won't break